assets/css/index.css

```diff
@@ -80,6 +83,7 @@ main {
     font-weight: 200;
     font-family: 'Inter', sans-serif;
     line-height: .75;
+    padding: 0 0 .25rem 0;
```

Comment:
How did you figure this out?

Reply:
I think I was googling "how to center text inside button... then i just messed around with the padding."